ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -417,6 +417,44 @@ default_interfaces:
         inter_domain: < true | false | Default -> True >
 ```
 
+### BGP & IP-VPN Gateway
+
+```yaml
+< node_type_key >:
+
+  defaults:
+    # Node is acting as IP-VPN Gateway for EVPN to MPLS-IP-VPN Interworking | Optional.
+    # The BGP peer group used for this is "bgp_peer_groups.mpls_overlay_peers".
+    # L3 Reachability is required for this to work, the preferred method to establish underlay connectivity is to use core_interfaces.
+    ipvpn_gateway:
+        enabled: < true | false | default -> False >
+
+        # Domain IDs are required to perform D-Path lookups for loop prevention. If omitted, the defaults are used.
+        evpn_domain_id: < "nn:nn" | default -> "65000:1" >
```

Comment:
I don't think we should have a default 65000 here. Either just 0:1 and 0:2 or no default at all.